helix-term/src/ui/mod.rs

```diff
@@ -95,7 +95,15 @@ pub fn regex_prompt(
 
                             view.ensure_cursor_in_view(doc, config.scrolloff);
                         }
-                        Err(_err) => (), // TODO: mark command line as error
+                        Err(_err) => {
+                            if event == PromptEvent::Validate {
+                                let error = format!("Invalid regex: {}", input);
```

Comment:
We should show the exact error returned by the regex library (formatting with `err` should work) since "Invalid input" isn't very useful for fixing the invalid regex.

Reply:
The regex errors are multiple lines, for example:
```
regex parse error:
    q(?!u)
     ^^^
error: look-around, including look-ahead and look-behind, is not supported
```
How should they be displayed? Through some pop up window? Or maybe shortened to fit on the one line of the status message?

Reply:
You could try creating a popup above the regex input and see how it looks.

Reply:
Added the popup, this is how it looks:
![image](https://user-images.githubusercontent.com/58790821/180206958-964df0a7-3ffe-4483-9851-6199e9976668.png)

Reply:
Popups are going to be annoying to clear, what about only taking the last line of the error for now? It contains a description

Reply:
How about making them auto close? That way they get cleared on the next keypress.